data/src/main/java/com/m3u/data/parser/epg/EpgParserImpl.kt

```diff
@@ -94,28 +94,44 @@ class EpgParserImpl @Inject constructor(
         var desc: String? = null
         val categories = mutableListOf<String>()
         var icon: String? = null
+        var isNewTag = false // Initialize isNew flag
+        var isLiveTag = false
+        var previouslyShownStart: String? = null // Initialize previouslyShown variable
         while (next() != XmlPullParser.END_TAG) {
             if (eventType != XmlPullParser.START_TAG) continue
             when (name) {
                 "title" -> title = readTitle()
                 "desc" -> desc = readDesc()
                 "category" -> categories += readCategory()
                 "icon" -> icon = readIcon()
+                "new" -> isNewTag = readNew() // Update isNewTag flag
+                "live" -> isLiveTag = readLive() // Update isNewTag flag
+                "previously-shown" -> previouslyShownStart = readPreviouslyShown()
                 else -> skip()
             }
         }
         require(XmlPullParser.END_TAG, ns, "programme")
+        // Add logic to determine whether the program is new based on title
+        val isNew = title?.contains("ᴺᵉʷ") ?: false
+        val isLive = title?.contains("ᴸᶦᵛᵉ") ?: false
```

Comment:
Is that a standard? 🤔

Reply:
its not xmltv standard but it is added by some providers instead of the <new /> and <live /> tags. this is just an additional method to find this data if its included, and the logic to decide looks at this and the tag and if either is true it will set it to true. this method will just support more users i think

Reply:
<previously-shown start="20220331180000 +0000" channel="channel-two.tv" />
    <premiere>First time on British TV</premiere>
    <last-chance lang="en">Last time on this channel</last-chance>
    <new />
these are all official xmltv tags which i added <previously-shown and <new />, will add the others.

Reply:
Not standard should not be adopted